Sources/ArgumentParser/Usage/HelpGenerator.swift

```diff
@@ -203,8 +214,16 @@ internal struct HelpGenerator {
           abstract: command.configuration.abstract)
     }
     
+    // Combine the compiled groups in this order:
+    // - arguments
+    // - named sections
+    // - options/flags
+    // - subcommands
```

Comment:
@rauhul @ethan-kusters Does this seem like the correct order?

Reply:
Yeah this makes sense to me!